src/Patcher.php

```diff
@@ -71,6 +75,21 @@ protected function patch(string $file, int $batch, bool $pretend): void
         $this->note("<info>Patched:</info>  {$name} ({$runTime} seconds)");
     }
 
+    /**
+     * Determine if patcher should run.
+     *
+     * @param \Illuminate\Database\Migrations\Migration $migration
+     * @return bool
+     */
+    public function isEligible(Migration $migration): bool
+    {
+        if (method_exists($migration, 'eligible')) {
+            return $migration->eligible();
+        }
+
+        return true;
+    }
+
```

Comment:
what is the use case of this method ?

Reply:
the one that u requested #1 🤦

Reply:
is this method supposed to called on line 66 ?
![image](https://user-images.githubusercontent.com/13056772/162562971-a6397ad1-b85c-4450-ac8a-e4ef3c46663c.png)